libs/game/game.ts

```diff
@@ -158,7 +158,7 @@ namespace game {
      * Finish the game and display the score
      */
     //% group="Gameplay"
-    //% blockId=gameOver block="game over || %win=toggleWinLose with %effect effect"
+    //% blockId=gameOver block="game over %gameWon=toggleWinLose || with %effect effect"
     //% weight=80 help=game/over
     export function over(win: boolean = false, effect?: effects.BackgroundEffect) {
```

Comment:
what happens when we decompile ?
```
game.over()
```

Reply:
It'll get a hole, same as if you decompile ``sprites.create(img`1`)``:

<img width="494" alt="Screen Shot 2019-04-15 at 4 59 53 PM" src="https://user-images.githubusercontent.com/5615930/56172791-efda2300-5f9f-11e9-8fab-ab7712b94188.png">

Reply:
That's unfortunate, ``game.over()`` is perfectly valid.

Reply:
Maybe for primitive types the decompiler should put in the correct default? I think that's safe enough

Reply:
Well, no, never mind. Didn't think that through

Reply:
Our api consistantely assume "false" as default so that should be good enough.

Reply:
We could at least detect primitives that have initializers in the function signatures. However, it might be a stretch to map it to the win/lose block since the XML for that might not be the same as a normal logic true/false

Reply:
https://github.com/Microsoft/pxt-arcade/issues/891